api/specs/common/schemas/node-meta-v0.0.1-pydantic-converted-clean.yaml

```diff
@@ -0,0 +1,589 @@
+additionalProperties: false
+description: 'Static metadata for a service injected in the image labels
+
+
+  This is one to one with node-meta-v0.0.1.json'
```

Comment:
do we really need all these yaml files in the repository?
as far as I understand these are not all used is that right?

Reply:
Previously: node-meta-v0.0.1.json --> node-meta-v0.0.1-converted.yaml
Current: node-meta-v0.0.1-pydantic.json --> node-meta-v0.0.1-pydantic-converted.yaml --> node-meta-v0.0.1-pydantic-converted-clean.yaml

Yes, the helper node-meta-v0.0.1-pydantic-converted.yaml can be potentially deleted, it is just helper to create the final node-meta-v0.0.1-pydantic-converted-clean.yaml that is used.

Reply:
in that case and if it can be re-generated I would not keep it in the repo.

Reply:
ok i have deleted all not used files: the helpers, but also the old json schema files. We still use them in the Director, but probably there is not need so they are in the api/v0/schemas folder anymore

Reply:
aha seems Director is complaining and wants to have the file there, we can discuss in person